.github/workflows/build_and_test.yml

```diff
@@ -1,4 +1,4 @@
-name: Build & Test utils_common
```

Comment:
As @thomas-moulard pointed out to me, you'll need to make sure the branch protection rules are up to date with any name changes to the actions.

Reply:
Yes, but I won't change it until after this PR is checked in